src/utils.ts

```diff
@@ -6,3 +6,28 @@ export function getElementById<T extends HTMLElement>(id: string): T {
     return ElementId as T;
   }
 }
+
+export class ToastManager {
+  constructor() {}
+
+  showAlertMessage(
+    cssClass: string,
```

Comment:
should be two methods `showSuccessMessage` and `showErrorMessage` which call private method with cssClass